### DISCUSSION TITLE
Azure China scope is incorrect

### DISCUSSION THREAD

**Discussion by bloomadcariad** [aquasecurity/trivy#7559] [Bugs]
### Description

In https://github.com/aquasecurity/trivy/commit/feaef9699df5d8ca399770e701a59d7c0ff979a3 https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/image/registry/azure/azure.go was modified, but the scope for China cloud was set incorrectly. Line 48 should be "chinaScope" but was copy-pasted as "scope." 

### Desired Behavior

Authentication with China cloud ACR should work.

### Actual Behavior

Authentication with China cloud ACR does not work.

### Reproduction Steps

```bash
1. Call trivy on an image in a China Cloud ACR
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
N/A
```

### Operating System

Ubuntu

### Version

```bash
N/A
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment]
Track https://github.com/aquasecurity/trivy/issues/7562
